### DISCUSSION TITLE
token-erc-721 - how to contribute golang version

### DISCUSSION THREAD

**Discussion by msalimbene** [hyperledger/fabric-samples#579] [General]
Hello,

I've translated token-erc-721 currently available only in JS, to golang. I'm finishing testing and cleaning up the code but I wanted to know how can I contribute it to the project.

Thanks.

**Comment by SamYuan1990** [comment]
I am not sure if you can create a PR and have online sharing as the ERC-721 session I know at https://wiki.hyperledger.org/display/fabric/Contributor+Meetings+2021
February 17, 2021
ERC-721 sample chaincode - Yuki Kondo

**Reply by msalimbene** [reply]
I just completed the port, fully tested and documented. But I cannot push the branch or create a PR. Should I join that meeting?

**Reply by SamYuan1990** [reply]
https://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html

... in short
fork
change code
`git commit -s`
push and PR back.

if you just want to create a PR...

**Comment by salimbene** [comment]
@SamYuan1990 thanks for your input.

I have a PR ready: https://github.com/hyperledger/fabric-samples/pull/625.

Do you know if I should let someone know to have it reviewed, or do I just wait?

**Reply by SamYuan1990** [reply]
@mbwhite , could you please help review the PR above?

**Comment by salimbene** [comment]
@SamYuan1990 @mbwhite  Thank you!
